src/app/helium.csproj

```diff
@@ -40,7 +40,7 @@
     <Content Update="appsettings.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Update="wwwroot\swagger.json">
+    <Content Update="wwwroot\swagger\helium.json">
```

Comment:
We should change this so that it copies the entire wwwroot directory instead of just one file. That way if people add additional files, it just works.